OWNERS

```diff
@@ -5,14 +5,12 @@
 approvers:
 - a-roberts
 - alangreene
+- eddycharly
 - mnuttall
 - skaegi
 - steveodonovan
 - dibbles
+- carolynmabbott
 
 reviewers:
-- akihikokuroda
-- carolynmabbott
-- megan-wright
-- ncskier
-- eddycharly
+
```

Comment:
Can we add an 'Alumni' section like in some of the other projects? https://github.com/tektoncd/pipeline/blob/0b345b56bda47c4542519102527889f7d0777a33/OWNERS_ALIASES#L30

Reply:
Great idea, added